packages/fxa-settings/src/pages/Signin/Signin/index.stories.tsx

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import Signin from '.';
+import AppLayout from '../../../components/AppLayout';
+import { LocationProvider } from '@reach/router';
+import { Meta } from '@storybook/react';
+import { EXAMPLE_EMAIL, EXAMPLE_SERVICE, EXAMPLE_ICON } from './mocks';
+export default {
+  title: 'pages/Signin/SigninPage',
+  component: Signin,
+} as Meta;
+
+// TODO: Add in error and success states when the Alert Bar is implemented
+
+export const PasswordNeeded = () => (
+  <LocationProvider>
+    <AppLayout>
+      <Signin
+        email={EXAMPLE_EMAIL}
+        isPasswordNeeded={true}
+        isPocketClient={false}
+        serviceName={EXAMPLE_SERVICE}
+      />
+    </AppLayout>
+  </LocationProvider>
+);
+
+export const PasswordNotNeeded = () => (
+  <LocationProvider>
+    <AppLayout>
+      <Signin
+        email={EXAMPLE_EMAIL}
+        isPasswordNeeded={false}
+        isPocketClient={false}
+        serviceName={EXAMPLE_SERVICE}
+      />
+    </AppLayout>
+  </LocationProvider>
+);
+
+export const IsPocketClient = () => (
+  <LocationProvider>
+    <AppLayout>
+      <Signin
+        email={EXAMPLE_EMAIL}
+        isPasswordNeeded={false}
+        isPocketClient={true}
+        serviceName={'Pocket'}
+      />
+    </AppLayout>
+  </LocationProvider>
+);
```

Comment:
Do we need a story for `true/true`?
We have one for `true/false`, `false/false`, and `false/true`, but not sure if we need `true/true` in the matrix.

Reply:
Sure! I'll add it in!

Reply:
So, I don't think we can require a password if it has Pocket as the client, but I did add in a story for when you have Pocket and also a logo (which I think should always be the case?) and added an example with a different logo for the false/true case. Thanks for the suggestion Peter!